optuna/integration/chainermn.py

```diff
@@ -25,6 +21,11 @@
     from typing import TypeVar  # NOQA
     from typing import Union  # NOQA
 
+    from optuna.distributions import BaseDistribution  # NOQA
+    from optuna.storages import BaseStorage  # NOQA
```

Comment:
Please remove this line because `BaseStorage` is not used in this file.

Reply:
Thanks.
Removed: https://github.com/pfnet/optuna/pull/427/commits/9b1828243a2145cbb419f78d682ae34e2fbc760c